sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java

```diff
@@ -179,6 +181,18 @@ public WritableColumnVector reserveDictionaryIds(int capacity) {
    */
   protected abstract void reserveInternal(int capacity);
 
+  /**
+   * Each byte of the returned value (long) has one bit from `bits`.
```

Comment:
Please expand on Javadoc. Although you explained how the method works, it is unclear what this method does, i.e. why we need it.

Reply:
Updated